### DISCUSSION TITLE
is possible debug in react native with devtools?

### DISCUSSION THREAD

**Discussion by misaku** [pmndrs/zustand#770] [Q&A]
is possible debug in react native with devtools?

**Comment by dai-shi** [comment]
I'm not super familiar, but if react-redux can, maybe zustand can too.

**Reply by misaku** [reply]
the problem is when I try to use, devtools or react-native-flipper-zustand I can't, and there seems to be no official way to do this, any suggestions?

**Reply by dai-shi** [reply]
Hmm, not sure. I wonder if anyone who has some experience can help.

**Reply by misaku** [reply]
i'm not sure if is possible at moment , but i think the idea is use remotle connection same remote-redux-dev tools,
i will put 2 libs, i try to understand and if i discover more or solve this problem i can contribute to this lib, but at moment i need help.

other libs 
https://github.com/zalmoxisus/remote-redux-devtools
https://github.com/jhen0409/remotedev-rn-debugger

both, combined use to connect rn to reduxdev tools

**Reply by misaku** [reply]
@dai-shi can you halp me? any idea?

**Reply by dai-shi** [reply]
I'm not so experienced with RN devtools. We need somebody to help here.

**Reply by misaku** [reply]
yah, i try many solution to solve this but not. if you help to use remote redux dev tool i tryed  in rn and nothing, now i tryed with web. i dont know but if i can resolve in web. i think to solution for RN  is not diff.

**Reply by ghost** [reply]
I used that and it's working: https://github.com/cmdominguez/flipper-plugin-zustandstore

You have to trigger a named action to see the action and the store content.

**Comment by g-otn** [comment]
See https://github.com/matt-oakes/redux-devtools-expo-dev-plugin/issues/11

**Comment by ilteoood** [comment]
I did a plugin the integrates with Rozenite: https://github.com/ilteoood/zorro
